Translate `Introduction to HTML and CSS` (#54)
kamp-us/turkce-odin-project: temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md

### Giriş

İşte Burada: Artık _bir şeyler yapmaya_ başlamanın zamanı geldi. Bu bölüm, internetteki neredeyse her şeyin temel yapı taşları olan HTML ve CSS'nin temellerini öğretecektir.

### Derse Genel Bakış

Bu bölüm, bu derste öğreneceğiniz konulara ilişkin genel bir bakış içerir.

- HTML, CSS ve iki teknolojinin birlikte nasıl çalıştığına ilişkin temel bir genel bakış edineceksiniz.

### HTML ve CSS

HTML ve CSS, internette bir şeye baktığınızda gördüğünüz her şeyi oluşturmak için birlikte çalışan iki dildir. HTML, bir internet sayfasının temel verilerini oluşturur. Tüm metinler, bağlantılar, kartlar, listeler ve düğmeler HTML'de oluşturulur. CSS ise bu sade öğelere _stil_ ekler. HTML, bilgiyi internet sitenize yerleştirir ve CSS bu bilgiyi konumlandırır, ona renk verir, yazı tipini değiştirir ve harika görünmesini sağlar!

> Birçok harika kaynak, HTML ve CSS'yi _programlama dilleri_ olarak sürekli bahseder, ancak teknik açıdan yaklaşmak istiyorsanız onları bu şekilde etiketlemek pek doğru değildir. Çünkü yalnızca bilgi sunma ile ilgilidirler. Herhangi bir mantığı programlamak için kullanılmazlar. Bir sonraki bölümde öğreneceğiniz JavaScript, bir programlama dilidir çünkü internet sitelerinin bir şeyler yapmasını sağlamak için kullanılır. Ancak sadece HTML ve CSS ile yapabileceğiniz pek çok şey var ve kesinlikle ikisine de ihtiyacınız olacak. Takip ettiğiniz dersler, müfredatımızda ilerledikçe JavaScript içeriğinde başarılı olmanızı sağlayacak araçları size sunmaktadır.

Reviewer: etiketlemek pek doğru değildir, çünkü yalnızca bilgi sunma ile ilgilidirler.

> Birçok harika kaynak, HTML ve CSS'yi _programlama dilleri_ olarak sürekli bahseder, ancak teknik açıdan yaklaşmak istiyorsanız onları bu şekilde etiketlemek pek doğru değildir. Çünkü yalnızca bilgi sunma ile ilgilidirler. Herhangi bir mantığı programlamak için kullanılmazlar. Bir sonraki bölümde öğreneceğiniz JavaScript, bir programlama dilidir çünkü internet sitelerinin bir şeyler yapmasını sağlamak için kullanılır. Ancak sadece HTML ve CSS ile yapabileceğiniz pek çok şey var ve kesinlikle ikisine de ihtiyacınız olacak. Takip ettiğiniz dersler, müfredatımızda ilerledikçe JavaScript içeriğinde başarılı olmanızı sağlayacak araçları size sunmaktadır.

Reviewer 2: ```suggestion
Birçok harika kaynak, HTML ve CSS'yi _programlama dilleri_ olarak sürekli bahseder, ancak teknik açıdan yaklaşmak istiyorsanız onları bu şekilde etiketlemek pek doğru değildir, çünkü yalnızca bilgi sunma ile ilgilidirler. Herhangi bir mantığı programlamak için kullanılmazlar. Bir sonraki bölümde öğreneceğiniz JavaScript, bir programlama dilidir çünkü internet sitelerinin bir şeyler yapmasını sağlamak için kullanılır. Ancak sadece HTML ve CSS ile yapabileceğiniz pek çok şey var ve kesinlikle ikisine de ihtiyacınız olacak. Bundan sonraki dersler, müfredatımıza devam ederken JavaScript içeriğine geldiğinizde başarılı olmanız için gereken araçları sağlamaya odaklanmaktadır!
```

Author: [ef20af8](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/ef20af861521c4e43a2b9fe3b8cf9c30e5eacfaa) Done!

### Ödev

<div class="lesson-content__panel" markdown="1">

> 1. Lütfen Okuyunuz [HTML ile CSS ile de Javascript Arasındaki Farklar](https://brytdesigns.com/html-css-javascript-whats-the-difference/). HTML, CSS ve JavaScript arasındaki ilişkinin kısa bir genel bakışıdır.

Reviewer 2: ```suggestion
1. [HTML vs CSS vs JavaScript](https://brytdesigns.com/html-css-javascript-whats-the-difference/) adlı ingilizce makaleyi okuyun. Bu, HTML, CSS ve JavaScript arasındaki ilişkinin kısa bir genel bakışıdır.
```

Author: [3b14e79](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/3b14e79cee29908931c42f9b8f432db9a4761f5c) done!

</div>

> ### Bilgi Ölçme Bakış

Reviewer 2: ```suggestion
### Bilgi Ölçme
```

Author: [064eab4](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/064eab4f649ae3c1be43ac110737eb6c946efafa) Done!

> Bu bölümde, bu dersi anladığınıza dair, kendinizi kontrol etmeniz için sorular bulunmaktadır. Bir soruyu yanıtlamakta sorun yaşıyorsanız, soruya tıklayın ve bağlantı verdiği materyali inceleyin.

Reviewer 2: ```suggestion
Bu bölüm, bu dersi kendi kendinize anlayıp anlamadığınızı kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta zorlanıyorsanız, soruya tıklayın ve bağlantılı olduğu materyali gözden geçirin.
```

Author: [064eab4](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/064eab4f649ae3c1be43ac110737eb6c946efafa) Done!

> - [HTML ve CSS ne anlamına geliyor?](https://brytdesigns.com/html-css-javascript-whats-the-difference/#What_is_HTML)

Reviewer 2: ```suggestion
- [HTML ve CSS ne anlamına geliyor? adlı ingilizce makale](https://brytdesigns.com/html-css-javascript-whats-the-difference/#What_is_HTML)
```

Author: [81602bc](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/81602bc297e5bc42208eeb73fe486927e17c6405) Done!

- [HTML ve CSS arasında, bir web sayfasında metin paragrafları eklemek için hangisini kullanırsınız?](#html-and-css)
- [HTML ve CSS arasında, bir düğmenin yazı tipini ve arka plan rengini değiştirmek için hangisini kullanırsınız?](#html-and-css)
- [HTML, CSS ve JavaScript arasındaki farklar nelerdir?](https://brytdesigns.com/html-css-javascript-whats-the-difference/)

### Ek Kaynaklar

Bu alanda içerikle alakalı faydalı linkler bulunmaktadır. Zorunlu değildir, ek olarak düşünülmelidir.

> - [FreeCodeCamp Makalesi](https://www.freecodecamp.org/news/html-css-and-javascript-explained-for-beginners/) belirlenenden biraz daha derine iner. Ancak daha sonraki derslerde açık bir şekilde öğreteceğimiz şeyleri kapsıyor. Bu nedenle ayrıntıları ezberleme konusunda endişelenmeyin.

Reviewer 2: ```suggestion
- Bu ingilizce [FreeCodeCamp Makalesi](https://www.freecodecamp.org/news/html-css-and-javascript-explained-for-beginners/) ödev olarak verilen makaleden biraz daha derine iniyor. Daha sonraki derslerde açık bir şekilde öğreteceğimiz şeyleri kapsıyor, bu nedenle herhangi bir ayrıntıyı ezberleme konusunda endişelenmeyin.
```

Author: [a31bbb5](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/a31bbb51d608f3497d1df36ad1481bff90cb54ff) done!

> - [DevDocs.io](https://devdocs.io) yer imlerinize ekleyiniz. "Hoş Geldiniz" mesajını okuyunuz. Massive API belgeleri koleksiyonu, çevrimdışı da kullanılabiliyor. Kapsanan her şey ve daha fazlası için temel bir başvuru materyali koleksiyonu. (Yöneten: [FreeCodeCamp](https://freecodecamp.org))

Reviewer 2: ```suggestion
- [DevDocs.io](https://devdocs.io)'yu yer imlerinize ekleyin. "Hoş Geldin" mesajını okuyun. Çevrimdışı da çalışan devasa API dokümantasyon koleksiyonu. Ele alınan her şey ve daha fazlası için temel bir referans materyal koleksiyonu. (Yöneten: [FreeCodeCamp](https://freecodecamp.org))
```

Author: [512643a](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/512643a3b0147e0ea932514271aabed65049c844) done!
